tools/config/__init__.py

```diff
@@ -769,9 +769,9 @@ def _make_header_region(self, start, header_format, offset=None):
         return (start, region)
 
     @staticmethod
-    def _assign_new_offset(rom_start, start, new_offset, region_name):
+    def _assign_new_offset(rom_start, start, new_offset, region_name, override=False):
```

Comment:
So:
 * Adding `override=True` skips the check in this function
 * This function does `rom_start + new_offset` and an overflow check

I think adding override to this function misses the point: this function exists for combining the overflow check with the new offset calculation.